### ISSUE TITLE
Upgrade to Laravel 5.2

### ISSUE THREAD

**Issue by austenpayan** [matthewerlemiller/projectgenesys#70]
Eventually, I will be upgrading our backend framework to Laravel 5.2 (currently at 4.2). Here is a screenshot of the server requirements for 5.2, which we may or may not already have.
![screen shot 2016-05-04 at 11 38 58 am](https://cloud.githubusercontent.com/assets/3936068/15025191/f139c272-11ec-11e6-9cd9-5a2a8194fcaf.png)

I won't be doing this immediately, but @matthewerlemiller can you look over this list to make sure we can have these ready to go for when I do?

**Comment by austenpayan** [comment]
Reason being : Laravel 5.2 uses modern syntax and PHP styling, as well as a few new features that would make development easier, its a bit faster, etc.

**Comment by matthewerlemiller** [comment]
Yes, let me get details before we move forward with this. I'll want to take
a great snapshot of the machine as well as choose a good time so there is
no downtime for YV.

I'll talk to you about this when I have some more details.
